assets/large_language_models/rag/components/src/chat_flow_with_varients_mlindex.json

```diff
@@ -0,0 +1,413 @@
+{
+  "flowName": "@@Flow_Name",
+  "flow": {
+    "flowGraph": {
+      "id": "@@Flow_Name",
+      "name": "@@Flow_Name",
+      "nodes": [
+        {
+          "name": "embed_the_question",
+          "tool": "Embedding",
+          "inputs": {
+            "connection": "@@Embedding_Connection",
+            "deployment_name": "@@Embedding_Deployment_Name",
+            "input": "${flow.question}"
+          }
+        },
+        {
+          "name": "search_question_from_indexed_docs",
+          "tool": "Vector Search",
+          "inputs": {
+            "path": "@@MLIndex_Asset_Id",
+            "query": "${embed_the_question.output}",
+            "top_k": "2"
+          },
+          "reduce": false
+        },
+        {
+          "name": "generate_prompt_context",
+          "tool": "generate_prompt_context",
+          "inputs": {
+            "search_result": "${search_question_from_indexed_docs.output}"
+          },
+          "reduce": false
+        },
+        {
+          "name": "Prompt_varients",
+          "tool": "Prompt_varients",
+          "inputs": {
+            "contexts": "${generate_prompt_context.output}",
+            "question": "${flow.question}",
+            "chat_history": "${flow.chat_history}"
+          },
+          "reduce": false
+        },
+        {
+          "name": "answer_the_question_with_context",
+          "tool": "answer_the_question_with_context",
+          "inputs": {
+            "deployment_name": "@@Completion_Deployment_Name",
+            "suffix": "",
+            "max_tokens": "1000",
+            "temperature": "0",
+            "top_p": "1.0",
+            "logprobs": "",
+            "echo": "False",
+            "stop": "",
+            "presence_penalty": "0",
+            "frequency_penalty": "0",
+            "best_of": "1",
+            "logit_bias": "",
+            "prompt_text": "${Prompt_varients.output}"
+          },
+          "provider": "AzureOpenAI",
+          "connection": "@@Completion_Connection",
+          "api": "@@API",
+          "module": "promptflow.tools.aoai",
+          "reduce": false
+        }
+      ],
+      "inputs": {
+        "question": {
+          "type": "string",
+          "is_chat_input": true
+        },
+        "chat_history": {
+          "type": "list",
+          "is_chat_history": true,
```

Comment:
I know "is_chat_input" is a thing but is "is_chat_history" also something that PF supports?

Reply:
I dont see it in the ES sample for MLIndex

Reply:
Strangely, the ES sample wasn't needing the flag, but this one did. I will add on ES side to be extra thorough.